ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -19,32 +19,12 @@ Use an ingress controller when you want to:
 * Give your Kubernetes app an externally-reachable URL.
 * Load-balance traffic to your app.
 
-Kubernetes provides an NGINX ingress controller that you can use in Docker EE
-without modifications.
-Learn about [ingress in Kubernetes](https://v1-8.docs.kubernetes.io/docs/concepts/services-networking/ingress/).
+Kubernetes provides an NGINX ingress controller that can be used in Docker EE, but is not officially supported. Modifications are typically required based on your environment.
+Learn about [ingress in Kubernetes](https://v1-11.docs.kubernetes.io/docs/concepts/services-networking/ingress/). 
 
 ## Create a dedicated namespace
```

Comment:
`Create a dedicated namespace` can be dropped entirely, and the YAML link moved down for pasting in section `Deploy NGINX ingress controller`.

Reply:
Will remove - thank you!